test/specs/scenarios/document.js

```diff
@@ -201,8 +201,8 @@ export const documentScenarios = async (lang) => {
         confirm.verifyUseAnotherFileError(copy)
       })
 
-      it('should return glare detected message on front and back of doc', async () => {
-        driver.get(`${baseUrl}&async=false&useUploader=true`)
+      it.skip('should return image quality message on front of doc', async () => {
```

Comment:
When are these skipped tests going to be enabled?

Reply:
Good question! There's a story for that: [CX-5545](https://jira.onfido.co.uk/browse/CX-5545) & it shall be done when the production APIs are fully enrolled with `image-quality` service.